torch_xla/csrc/ir.h

```diff
@@ -122,15 +122,17 @@ class XlaNode : public torch::lazy::Node {
   torch::lazy::hash_t shardingHash() const { return sharding_hash_; }
 
   // The node's outputs get assigned the same HLO sharding
-  // TODO: test multi-output example.
-  const std::shared_ptr<xla::OpSharding> GetSharding() const {
-    return output_sharding_;
+  const std::shared_ptr<xla::OpSharding> GetSharding(size_t index) const {
+    if (output_shardings_.size() == 0) {
```

Comment:
index is not used.

Reply:
my bad.. it should return `output_shardings_[index]`. My test is not general enough, let me add a new one.